gravitee-apim-portal-webui-next/src/components/button/button.scss

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2024 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@use 'sass:map';
+@import '../../scss/theme/theme';
+
+$button-primary-outline-color: map.get($mat-primary, main);
+
+button.mdc-button {
+  padding: 10px 24px;
+  border-radius: 100px;
+  letter-spacing: 0.1px;
```

Comment:
Instead of "letter-spacing", you can use the token `--mdc-protected-button-label-text-tracking`